ignite/distributed/__init__.py

```diff
@@ -0,0 +1 @@
+from ignite.distributed.utils import *
```

Comment:
I did this during prototyping. I think it’s not a good idea to import all functions (and it’s not done in ignite).

Reply:
We have `__all__` inside `ignite.distributed.utils`. I think it is OK.

Reply:
> Do you suggest to import manually what is helpful ?
Similar to others modules. But that is not really important. I thought it was not a good practice to import all.